examples/inputs/example_01d/lines.csv

```diff
@@ -1,2 +1,2 @@
-name,bus0,bus1,s_nom,x,r
-0,north,south,5000,0.01,0.001
+name,bus0,bus1,s_nom,x,r,shape
+0,north,south,5000,0.01,0.001,"LINESTRING (9.598 53.5585, 13.607 51.0769)"
```

Comment:
So @nick-harder you would rather have lon and lat in buses/powerplants and let assume create an according `LINESTRING` from the bus0 and bus1 connection on the go instead of here?
Sounds good too.

Then we do not need to have this information in every file

Reply:
yes exacty, only the coordinates in the busses should be enough to construct the whole thing.